src/components/HomePage/ContactSection.spec.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import ContactSection from "./ContactSection";
+import ReactDOM from "react-dom";
+import { render, cleanup } from "@testing-library/react";
+// import HomePage from "../../pages/HomePage";
+
+afterEach(cleanup);
```

Comment:
The rtl docs state the following:

> Please note that this is done automatically if the testing framework you're using supports the afterEach global and it is injected to your testing environment (like mocha, Jest, and Jasmine). If not, you will need to do manual cleanups after each test.

So you should remove this line to avoid confusion in the future